### PR TITLE
Display personal contract details

### PR DESCRIPTION
**Pull request by huwshimi**
## Done

- Display personal contract details.
- Add fetching of personal subscriptions and types.
- Remove enterprise contracts.

## QA

- Visit [/advantage?test_backend=true](https://ubuntu-com-10197.demos.haus/advantage?test_backend=true).
- You should see the [free account details](https://app.zeplin.io/project/60ec5fb4e07cc90fbbf2b318/screen/60edc152de9015ae8a35ccd2) on the right (but not in the tabs at the bottom, that'll be done [later](https://github.com/canonical-web-and-design/commercial-squad/issues/134)).

## Issue / Card

Fixes https://github.com/canonical-web-and-design/commercial-squad/issues/118.

## Screenshots

Screen Shot 2021-08-27 at 1 31 21 pm

### CODE REVIEW

**Comment by huwshimi** [comment]
Blocked on API changes.

File: static/js/src/advantage/react/components/Subscriptions/SubscriptionDetails/DetailsContent/DetailsContent.tsx [hunk 1]
```diff
@@ -4,54 +4,75 @@ import {
...
+      </p>
     </Col>
   ));
 
 const DetailsContent = () => {
+  const { data: subscription, isLoading } = useUserSubscriptions({
+    // TODO: Get the selected subscription once the subscription token is available.
+    select: selectFreeSubscription,
+  });
+  const isFreeSubscription = subscription?.type === UserSubscriptionType.Free;
```
Comment (ClementChaumel) [inline static/js/src/advantage/react/components/Subscriptions/SubscriptionDetails/DetailsContent/DetailsContent.tsx:43]: Here the subscription returned will always be the free one as specified in the select. So I don't see the point of the check right after.
I assume you plan on iterating on every subscription later on and remove the select?

**Review by ClementChaumel — APPROVED** [review]
Looking good! :+1:

File: static/js/src/advantage/react/components/Subscriptions/SubscriptionDetails/DetailsContent/DetailsContent.tsx (hunk 1 above)
Comment (huwshimi) [reply static/js/src/advantage/react/components/Subscriptions/SubscriptionDetails/DetailsContent/DetailsContent.tsx:43]: Yeah that's right, the token from the selected sub from the subs list will be stored in state. At the moment the API doesn't return the token so there's no way of identifying the subs so I'm just selecting the free one for now.

**Pull request merged**